src/rtctools_heat_network/pycml/component_library/heat/ates.py

```diff
@@ -38,6 +39,10 @@ def __init__(self, name, **modifiers):
         self.rho = 988.0
         self.Heat_nominal = self.cp * self.rho * self.dT * self.Q_nominal
 
+        self.add_variable(Variable, "Temperature_ates", nominal=self.T_return)
+        self.add_variable(Variable, "Temperature_loss", min=0, max=1, nominal=self.T_supply/3600)
```

Comment:
I think the bounds are incorrect here, should be min=-1, max=0. I suppose

I think the nominal of T_supply/3600 is way too high, I would expect something in the order of a temperature change over a half year.

Reply:
improved, as well as the bounds